assets/scss/settings/fonts.scss

```diff
@@ -0,0 +1,11 @@
+$Georgia : Georgia;
```

Comment:
Name of variable `Georgia` should be written in all lowercase letters with hyphens instead of underscores<br>Variable names should be followed immediately by a colon